contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsRequest.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+interface IFunctionsRequest {
+  struct Commitment {
+    uint96 adminFee; // -----------┐
+    address coordinator; // -------┘
+    address client; // ------------┐
+    uint64 subscriptionId; //      |
+    uint32 callbackGasLimit; // ---┘
+    uint96 estimatedTotalCostJuels; // TODO pack the following
```

Comment:
Are these comments still relevant or can they be removed?

Reply:
Yes, still relevant. Still need to pack these more efficiently. I'll do it in another PR after running it by @RensR